pennylane_lightning/lightning_qubit/lightning_qubit.toml

```diff
@@ -2,22 +2,22 @@ schema = 2
 
 [operators.gates.native]
 
-CNOT                   = { properties = [ "invertible",                 "differentiable" ] }
-ControlledPhaseShift   = { properties = [ "invertible",                 "differentiable" ] }
-ControlledQubitUnitary = { properties = [ "invertible",                                  ] }
-CRot                   = { properties = [ "invertible"                                   ] }
-CRX                    = { properties = [ "invertible",                 "differentiable" ] }
-CRY                    = { properties = [ "invertible",                 "differentiable" ] }
-CRZ                    = { properties = [ "invertible",                 "differentiable" ] }
-CSWAP                  = { properties = [ "invertible",                 "differentiable" ] }
-CY                     = { properties = [ "invertible",                 "differentiable" ] }
-CZ                     = { properties = [ "invertible",                 "differentiable" ] }
+CNOT                   = { properties = [ "invertible", "controllable", "differentiable" ] }
```

Comment:
Unfortunately non of the aliased control gates are controllable from a strict point on view, so the previous version is correct I think.

Reply:
yeah, it's related to my other branch removing the C(op) gates from `_operations` and it's outdated  ;) Removed them all here.